### PR TITLE
Visualizations for cash bail rate and race breakdown

### PR DESCRIPTION
**Pull request by alexwgraves**
## Changes

- I added GT America, the font that these will be in on the website. For visualizations in particular this is important to ensure the sizing and placement is correct with the CSS.
  - Rong and I figured out a workaround for the condensed face, which didn't seem to be working. I've add in this PR the file referenced at the top of `table.css`. You should be able to install the regular one (i.e. not condensed) and have the rest of the text work fine — this seems to be an issue with how the font file is constructed for the `font-stretch` property.
- This PR creates the `BarGraphCell` and `NumberLineCell` for the visualizations. The former should only ever take one number, whereas the second can display up to two.
  - We'll be able to reuse `NumberLineCell` over a bunch of the graphs, and `BarGraphCell` can probably be extended for the bail type distribution tables
  - These are configured by specifying the `type` of the visualization when you pass in the data array: `bar` for `BarGraphCell` and `line` for `NumberLineCell`.
- I also created the `VizHeaderCell` which extended `HeaderCell` and displays the ticks and numbers for the start and end of the graphs as well as for the average(s)
  - There are some issues with the spacing for this, particularly line 197 of `Table.js`. I had to adjust the padding manually until the ticks lined up with the average line in the cells. I'm going to keep looking into this, but for the time being those hand-coded values are relative to the width of the text box, which isn't ideal.
- I added initial styling for mobile in `mobile.css`. It's not perfect, but everything fits without horizontal scrolling on a modern-sized phone. (Think iPhone X, Pixel, but not like an iPhone SE or one of the smaller ones. They can deal with some scroll.)

### CODE REVIEW

File: src/js/classes/Table.js [hunk 1]
```diff
@@ -101,6 +167,63 @@ class HeaderCell extends Cell {
...
+      const className = "average";
+      return this.createTickElement(text, className, vizColors[i]);
+    });
+    // create wrapper around averages for positioning
+    const averageWrapper = document.createElement("div");
+    averageWrapper.className = "average-wrapper";
+    // offset the average elements by the value/end ratio (and subtract padding)
+    averageElements.forEach((element, i) => {
+      const left = averages[i]["value"] / end * 100;
+      const padding = averages.length > 1 ? "0.45em" : "2.05em";
```
Comment (apothuku) [inline src/js/classes/Table.js]: This is **super** nitpicky and something you don't have to fix right now but the "State Average" tick is like .1em to the left of the average line. Hopefully this isn't a difference in our two screens. The White/Black tics look correct on my screen.

**Comment by apothuku** [comment]
I don't know enough CSS to give that portion a meaningful review. But I read it and it looks good to me.

Everything else looks fine - feel free to merge if you don't want to deal with the tick that's slightly-off on my screen (it might look fine on your screen, which brings up the scary possibility that it could look different on different screens)

File: src/js/classes/Table.js (hunk 1 above)
Comment (alexwgraves) [reply src/js/classes/Table.js]: Fixed this! What I was doing before was super imprecise, but by adding some transforms, I was able to do this solidly all corresponding to hard-coded numbers.

**Comment by apothuku** [comment]
The tick is fixed on my end! This is good to merge whenever you'd like

**Pull request merged**